### PR TITLE
Issue #122 - Usability improvements

### PR DESCRIPTION
**Pull request by armenzg**

### CODE REVIEW

**Comment by armenzg** [comment]
Link to deployment:
https://platform-health-pr-129.herokuapp.com/android

Current look:
image

**Comment by armenzg** [comment]
This is ready for review. Thanks Will for all your help with this!

image

**Comment by armenzg** [comment]
@wlach as I mentioned on IRC. This is ready to be reviewed.

File: src/components/StatusWidget/index.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+
+const StatusWidget = ({
+  children, extraInfo,
+  title,
+  summary, uid,
+}) => {
+  const { bgColor, emphasis, hyperlink, text, tooltip } = title;
+  let $title = '';
```
[the remark this answers is not in the document]
Comment (armenzg) [reply src/components/StatusWidget/index.jsx]: `$element` is something I've only seen in this repo. It's only been good to make the render methods a little less crowded and more clearly see the overall tree of divs.

File: src/views/Android/index.jsx [hunk 2]
```diff
@@ -38,22 +57,116 @@ export default class Android extends Component {
...
-          {Object.keys(nimbledroidData).length > 0 &&
-            <NimbledroidGraphs
-              nimbledroidData={nimbledroidData}
-            />
-          }
-          {errorMessage &&
-            <h2 style={{ color: 'red' }}>{errorMessage}</h2>
-          }
+        <div style={{ backgroundColor: 'white', color: 'black', padding: '1em' }}>
+          <div className='aligned-center'>
```
[the remark this answers is not in the document]
Comment (armenzg) [reply src/views/Android/index.jsx]: I explained this in the comment.

**Comment by armenzg** [comment]
Hi @wlach 
I've addressed all your feedback. 

If it helps, I have gathered all the changes from today in a [single commit](https://github.com/mozilla/firefox-health-dashboard/pull/129/commits/cc28bd2337c9c2ec5ca272ae132d2b1c4fde996a).

File: src/utils/nimbledroid/index.jsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
+const percentageSymbol = (GV, WV, targetRatio) => ((GV < (targetRatio * WV)) ? '+' : '');
+
+const ratioWithTarget = (GV, WV, targetRatio) => GV / (targetRatio * WV);
+
+export const sortSitesByTargetRatio = (a, b) => {
+  const aRatio = a.GV / a.WV;
+  const bRatio = b.GV / b.WV;
+  let retVal;
```
Comment (wlach) [inline src/utils/nimbledroid/index.jsx]: The rest of this function could be simplified to be just `return bRatio - aRatio` 😄

**Pull request merged**